cumulusci/tasks/bulkdata/mapping_parser.py

```diff
@@ -0,0 +1,60 @@
+from typing import Dict, List, Union, IO
+from logging import getLogger
+from pathlib import Path
+
+from pydantic import Field, validator, ValidationError
+
+from cumulusci.utils.yaml.model_parser import CCIDictModel
+
+LOGGER_NAME = "MAPPING_LOADER"
+logger = getLogger(LOGGER_NAME)
+
+
+class Lookup(CCIDictModel):
+    "Lookup relationship between two tables."
+    table: str
+    key_field: str = None
+    value_field: str = None
+    join_field: str = None
+    after: str = None
+    aliased_table: str = None
+
+
+class Step(CCIDictModel):
```

Comment:
Might be worth pseudo-namespacing this as something like `MappingStep`. We didn't use `Step` for Bulk API jobs due to genericity.

Reply:
I'll change this but it should be fairly infrequent to need to import intermediate grammar node names (except in tests, maybe).

Reply:
207aaeabc41b